README.md

```diff
@@ -91,6 +93,88 @@ Displays all of these options
 lighthouse-parade https://cloudfour.com --exclude-path-glob "/thinks/*" --max-crawl-depth 2 --output cloudfour-a.csv
 ```
 
+### Config File
+
+A config file can be specified via `-c` or `--config`. It must be a JS file that provides a default export that is the configuration object.
+
+Example `config.mjs`:
+
+```js
+// @ts-check
+
+import { defaultCrawler, defineConfig } from 'lighthouse-parade';
+
+export default defineConfig({
+  outputs: [{ type: 'csv', name: 'cloudfour.csv' }],
```

Comment:
Seeing this made me wonder if it might be confusing to remember, does the name include the extension? Does it need an extension? 

I could see being unsure if this:
```js
{ type: 'csv', name: 'cloudfour' }
```
Or this:
```js
{ type: 'csv', name: 'cloudfour.csv' }
```

Is there a world where we only pass the name (with extension) and the code pulls the extension off to figure out the `type`?

Or should the `name` not include the extension?

I'm not sure. These are just thoughts that hit me as I saw this.

Reply:
@gerardo-rodriguez Should I call the property `path` instead for CSV outputs? Would that make it more clear that it should include the extension?

Reply:
@calebeby Good question. I'm not sure…

Is `filename` too off-base?

Curious if @spaceninja or @Paul-Hebert have any thoughts on this. 🤔

Reply:
`filename` works 👍